### ISSUE TITLE
Switch to hand pointer cursor when hovering on interactive features

### ISSUE THREAD

**Issue by bcsheesley** [axismaps/rice#13]
E.g., buildings, streets. I bet this isn't really possible, but thought I'd check.

**Comment by awoodruff** [comment]
I don't think we can do it, but on a related note, should we have a Rio-esque "click the map to explore" tip when you first load the map?

**Comment by bcsheesley** [comment]
Yeah, good idea. Maybe something like this? It could go away after the first click/tap.

Related: It feels a little odd to probe with the open hand. Should we default to the arrow mouse pointer and change to the hand (or move cursor) on mouse down for panning? If not, I could probably live with it.

![desktop-hint](https://user-images.githubusercontent.com/4501388/33346031-e4f4c3ba-d453-11e7-9bd2-c4f5a0af773c.png)

![mobile-hint](https://user-images.githubusercontent.com/4501388/33346033-e80ecd98-d453-11e7-8894-784ce5dce165.png)

![mobile-hint-alt](https://user-images.githubusercontent.com/4501388/33346038-ea46900a-d453-11e7-8e1b-331bf67750bc.png)

**Comment by awoodruff** [comment]
Didn't mean to close this. Still needs icons.

**Comment by bcsheesley** [comment]
Can I make an annoying request and ask for that gray background color to be #333 and 0.9 opacity instead of what I have in the screenshots above? I think that'll be less dull and stand out better.

For example:
![screen shot 2017-12-14 at 5 58 06 pm](https://user-images.githubusercontent.com/4501388/34020250-c5526fec-e0f8-11e7-8f83-3e6428f1d756.png)
